new/PRS13304GabAm.xml

```diff
@@ -0,0 +1,73 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13304GabAm" xml:lang="en" type="pers">
```

Comment:
Here again: you also have this file in the other PR (#303). So in this case, the PR #303 can be simply deleted, both new person records (PRS13304GabAm and PRS13306BaMaryam) are unified in this PR.

@thea-m, @PietroLiuzzo  This would be the simpliest solution, right?

Reply:
perhaps delete also the branch there since that is not used any more.